traffic_portal/app/src/common/modules/header/header.tpl.html

```diff
@@ -80,9 +80,34 @@
                     <button type="button" class="btn btn-link" ng-if="hasCapability('cdns-snapshot')" ng-click="snapshot()"><i class="fa fa-camera"></i></button>
                 </div>
             </li>
-            <li ng-repeat="cdn in cdns">
-                <div class="btn-group" title="{{(cdn.notificationCreatedBy) ? 'Delete Global ' + cdn.name + ' Notification' : 'Create Global ' + cdn.name + ' Notification'}}">
-                    <button type="button" class="notification btn btn-link" ng-click="toggleNotification(cdn)">{{cdn.name}} <i class="fa" ng-class="{ 'fa-bell': cdn.notificationCreatedBy, 'fa-bell-o': !cdn.notificationCreatedBy }"></i> <i ng-show="cdn.notificationCreatedBy" class="fa fa-times"></i></button>
+            <li role="presentation" class="dropdown" ng-repeat="cdn in cdns | orderBy: '-name'">
+                <div class="btn-group" title="{{cdn.name}} Notifications" uib-dropdown is-open="{{cdn.name}}-notifications.isopen">
```

Comment:
button should have `type=button` since it isn't part of a `form`

Reply:
the button on line 85 has type=button already.

Reply:
My bad, missed it.